Android/app/src/main/java/io/github/project_travel_mate/utilities/CompassActivity.java

```diff
@@ -29,9 +35,35 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         setTitle("Compass");
         Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
         getSupportActionBar().setHomeButtonEnabled(true);
+
+        //Check accelerometer sensor for device
+        getAccelerometerSensor();
         //Setup compass
         setupCompass();
     }
+
+
+    /**
+     * Check AccelerometerSensor in device
+     **/
+    private void getAccelerometerSensor() {
+        PackageManager mManager = getPackageManager();
+        boolean hasAccelerometer = mManager.hasSystemFeature(PackageManager.FEATURE_SENSOR_ACCELEROMETER);
+        boolean hasMagneticSensor = mManager.hasSystemFeature(PackageManager.FEATURE_SENSOR_COMPASS);
+        if (!hasAccelerometer || !hasMagneticSensor) {
+            AlertDialog alertDialog = new AlertDialog.Builder(CompassActivity.this).create();
+            alertDialog.setTitle("Info");
```

Comment:
Please don't add contant strings in java code. Add them to strings.xml file & use those here